Sources/joyfill-components-swift/Components/Table/ViewTable.swift

```diff
@@ -398,46 +449,93 @@ public class ViewTable: UIViewController, TextFieldCellDelegate, TextViewCellDel
         }
     }
     
+    // Empty cell data to append or insert when insertRow and insertBelow is tapped
+    var data = [String]()
+    func emptyCellsData() {
```

Comment:
I would change this to insertEmptyCellsData - as the name doesn't tell me that below when we use it it's for setting up a new empty row when a user taps Add Row Below

Reply:
Done.